src/CentreonRemote/Tests/Infrastructure/Export/ExportParserJsonTest.php

```diff
@@ -75,10 +85,10 @@ public function testParse2()
     /**
      * @covers \CentreonRemote\Infrastructure\Export\ExportParserJson::parse
      */
-    public function testParse3()
+    public function testParse3(): void
     {
         // add file with macros
-        $this->fs->get('/tmp')->add('test2.json', new File('{"key":"@val@"}'));
+        $this->fs->get('/tmp')->add('test2.json', new File('{"key":"@val@"}')); /** @phpstan-ignore-line */
```

Comment:
To be discussed

Reply:
we can ignore this error cause we'll don't waste time to refactor this test